feature/org.wso2.carbon.identity.casque.authenticator.feature/org.wso2.carbon.identity.casque.authenticator.iml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need to commit a ".iml" file?

Reply:
fixed with - https://github.com/wso2-extensions/identity-outbound-auth-casque-snr/pull/8/